src/app/features/location/address-edit/address-edit.component.html

```diff
@@ -65,7 +65,13 @@
 </div>
 
 @if (!disabled) {
-  <app-address-search
-    (locationSelected)="updateFromAddressSearch($event)"
-  ></app-address-search>
+  <div class="margin-bottom-regular container">
+    <div class="flex-row gap-regular align-start full-width">
```

Comment:
is there a benefit from having two divs rather than one here?

Reply:
I refactored the code by eliminating an extra div and combining its class with another div.